data/data/aws/vpc/master-elb.tf

```diff
@@ -1,38 +1,44 @@
 resource "aws_lb" "api_internal" {
-  name                             = "${var.cluster_name}-int"
+  name                             = "${var.cluster_id}-int"
   load_balancer_type               = "network"
   subnets                          = ["${local.private_subnet_ids}"]
   internal                         = true
   enable_cross_zone_load_balancing = true
   idle_timeout                     = 3600
 
-  tags = "${var.tags}"
+  tags = "${merge(map(
+    "Name", "${var.cluster_id}-int",
+  ), var.tags)}"
 
   depends_on = ["aws_internet_gateway.igw"]
 }
 
 resource "aws_lb" "api_external" {
-  name                             = "${var.cluster_name}-ext"
+  name                             = "${var.cluster_id}-ext"
   load_balancer_type               = "network"
   subnets                          = ["${local.public_subnet_ids}"]
   internal                         = false
   enable_cross_zone_load_balancing = true
   idle_timeout                     = 3600
 
-  tags = "${var.tags}"
+  tags = "${merge(map(
+    "Name", "${var.cluster_id}-ext",
+  ), var.tags)}"
 
   depends_on = ["aws_internet_gateway.igw"]
 }
 
 resource "aws_lb_target_group" "api_internal" {
-  name     = "${var.cluster_name}-api-int"
+  name     = "${var.cluster_id}-api-int"
   protocol = "TCP"
   port     = 6443
   vpc_id   = "${local.vpc_id}"
 
   target_type = "ip"
 
-  tags = "${var.tags}"
+  tags = "${merge(map(
+    "Name", "${var.cluster_id}-api-int",
```

Comment:
Why set `Name` on resources that have a `name` property, if we're setting the same value in both?  I think it only makes sense when we feel like `name` must be overly truncated and want the tag for the unabbreviated version.

Reply:
from https://github.com/openshift/installer/pull/1280/commits/d6de59aa39811ee0511998a8d251837026ac8ef5
```
- All tags include `Name` key with name mathcing the name for the resource.
This allows cluster to find resources using tags based on names.
```